Fix error handling on duplicated APNS devices

bielfrontera opened the pull request:
If we have a duplicated registration ID on the APNSDevice model and we get an unregistered error after sending a message, it raises a `MultipleObjectsReturned` exception (get() returned more than one APNSDevice -- it returned ...!).

In order to fix this, we can update all the records with that registration ID instead of using `objects.get`.

jamaalscarlett commented:
Thanks @bielfrontera. It looks like there is a merge conflict, once that is fixed we can get this merged.

bielfrontera commented:
> Thanks @bielfrontera. It looks like there is a merge conflict, once that is fixed we can get this merged.

Great! I've rebased and fixed the merge conflict.
